src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp

```diff
@@ -896,6 +896,55 @@ void jit_select_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, const
     h->mov(dst.b16, aux.b16);
 }
 
+// SoftPlus
+jit_soft_plus_emitter::jit_soft_plus_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
+                                             dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                             const std::shared_ptr<ov::Node>& node)
+                                             : jit_emitter(host, host_isa, node, get_arithmetic_binary_exec_precision(node)) {
+}
+jit_soft_plus_emitter::jit_soft_plus_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
+                                             dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                             const ov::element::Type exec_prc)
+                                             : jit_emitter(host, host_isa, exec_prc) {
+}
+size_t jit_soft_plus_emitter::get_inputs_count() const { return 1; }
+size_t jit_soft_plus_emitter::get_aux_vecs_count() const { return 1; }
+
+std::set<std::vector<element::Type>> jit_soft_plus_emitter::get_supported_precisions(const std::shared_ptr<ov::Node>& node) {
+    return {{element::f32}};
+}
+
+void jit_soft_plus_emitter::emit_impl(const std::vector<size_t>& in_vec_idxs, const std::vector<size_t>& out_vec_idxs) const {
+    if (host_isa_ == dnnl::impl::cpu::aarch64::asimd) {
+        emit_isa<dnnl::impl::cpu::aarch64::asimd>(in_vec_idxs, out_vec_idxs);
+    } else {
+        OV_CPU_JIT_EMITTER_THROW("Can't create jit eltwise kernel");
+    }
+}
+
+template <dnnl::impl::cpu::aarch64::cpu_isa_t isa>
+    void jit_soft_plus_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
+        OV_CPU_JIT_EMITTER_ASSERT(exec_prc_ == ov::element::f32, "unsupported precision: " + exec_prc_.to_string());
+        using TReg = typename dnnl::impl::cpu::aarch64::cpu_isa_traits<isa>::TReg;
+        const TReg src = TReg(in_vec_idxs[0]);
+        const TReg dst = TReg(out_vec_idxs[0]);
+        const TReg aux = TReg(aux_vec_idxs[0]);
+        h->vldr(src.d, v0.d);
+        h->vmul(dst.d, v0.d, v0.d);
+        h->vcvt(dst.s, dst.d);
+        h->vfma(dst.s, v0.s, vptrue, v1.s, dst.s);
+        h->vmaxnm(dst.s, dst.s, vmmin.s);
+        h->vcvt(dst.d, dst.s);
+        h->vexp2(dst.d, dst.d);
+        h->vadd(dst.d, dst.d, vone.d);
+        h->vrcps(dst.d, dst.d);
+        h->vrsqrts(dst.d, dst.d);
+        h->vadd(dst.d, dst.d, vzero.d);
+        h->vst1(dst.d, ptr[dst_reg]);
```

Comment:
1. You don't need to load anything in `emitter`. By the main concept: everything has been loaded in vector registers already, use registers.
2. Inside emitter you can use only `in_vec_idxs`, `out_vec_idxs` and `aux_vec_idxs` vector registers (their indexes).
3. In `float` implementation use `word` only.
4. Use instrucations from SIMD&FP Instructions from ARM64 instruction set. We don't support ARM32 in OpenVINO snippets.

Note, please, you can use PRs as example:
* [Abs](https://docs.openvino.ai/2023.3/openvino_docs_ops_arithmetic_Abs_1.html) operation: https://github.com/openvinotoolkit/openvino/pull/23692
* [Sigmoid](https://docs.openvino.ai/2023.3/openvino_docs_ops_activation_Sigmoid_1.html) operation, which reuses [Exp emitter](https://github.com/openvinotoolkit/openvino/blob/4ef40f0f0e70abf04f13310f3e7a8f6784667dbf/src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp#L820): https://github.com/openvinotoolkit/openvino/pull/23065

Reply:
Hey @eshoguli please tell me more details about this, I am busy with other things